src/app/_components/VoteForm/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ChangeEvent, useState } from 'react';
+
+import Icon from '@/src/components/Icon';
+import { cn } from '@/src/utils/cn';
+
+const VoteForm = () => {
+  const [options, setOptions] = useState<string[]>(['', '']);
+
+  const handleOptionChange = (
+    index: number,
+    event: ChangeEvent<HTMLInputElement>,
+  ) => {
+    const newOptions = [...options];
+    newOptions[index] = event.target.value;
+    setOptions(newOptions);
+  };
+
+  const handleAddOption = () => {
+    if (options.length === 6) return;
+    setOptions([...options, '']);
+  };
+
+  const handleRemoveOption = (index: number) => {
+    if (options.length === 2) return;
+    const newOptions = [...options];
+    newOptions.splice(index, 1);
+    setOptions(newOptions);
+  };
+
+  return (
+    <div className='flex flex-col gap-1 rounded-md bg-dark-accent1 p-2'>
+      {options.map((option, index) => (
+        <div
+          key={`${index}`}
+          className='flex w-full items-center gap-2 bg-dark-accent1 p-2'
+        >
+          <input
+            value={option}
+            onChange={event => handleOptionChange(index, event)}
+            className='w-full rounded-sm border border-gray-accent2 bg-dark-accent1 p-3 text-black text-gray-accent1 outline-none ring-gray-accent1 focus:ring-2'
```

Comment:
```suggestion
            className='w-full rounded-sm border border-gray-accent2 bg-dark-accent1 p-3 text-gray-accent1 outline-none ring-gray-accent1 focus:ring-2'
```
text css 가 중복되어보입니다!

Reply:
text-black 이 왜있는지 모르겠네요 삭제하겠습니다.